server/validation/studentSchema.js

```diff
@@ -5,8 +5,7 @@ const studentSchema = yup.object({
   email: yup.string().email().required(),
   imgUrl: yup.string().url().required(),
   githubLink: yup.string().url().required(),
-  cohortId: yup.number().required(),
-  studentId: yup.number().required(),
```

Comment:
unfortunately the deleted line here is required for another function in order to work properly with perfect error handling, maybe if you remove `required()` it would work perfectly with your purpose without any problems.